README.md

```diff
@@ -2,9 +2,72 @@
 
 ABDM integrator is a REST API service packaged as a django app. 
 It implements all the three milestones of [ABDM](https://sandbox.abdm.gov.in/abdm-docs/getting-started).
+This package is intended to be used by developers aiming to perform integration with ABDM using python.
 
 ## Requirements
-
 - python >= 3.8
 - django >= 3.2
 - djangorestframework >= 3.12
+- requests
+
+## Build from source
+
+1. Clone the repository.
+2. Install python build. (If not already existing)
+    ```commandline
+     pip install build
+    ```
+3. Run below command to build.
+    ```commandline
+    python -m build
+    ```
+The resulting package (wheel or tar) can be then installed using pip into your Django Project.
```

Comment:
Is the plan to add this as a requirement in HQ side via github URL in the requirements.in?

Reply:
Correct.
```
# India - Used for ABDM Integration
abdm_python_integrator @ git+https://github.com/dimagi/abdm-python-integrator.git
```

If /when the package is published to PyPi is finalized, the github url would be removed.

Reply:
Addressed in 1b4607b3db43a4b77339c42836d9d510bd8ebb8d